RobotCode/src/main/java/frc/robot/constants/ElevatorConstants.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.constants;
+
+public class ElevatorConstants {
+	
+	public static final double[] ELEVATOR_POSITIONS = { // In encoder ticks (2048 ticks = 1 deg)
+		
+		25*2048,
```

Comment:
Make a separate constant variable for each position and name each according to the positions the elevator would need to reach in a match